#29: ExaLoader in one container imports from another container (#31)
exasol/exasol-testcontainers: doc/user_guide/user_guide.md

The simplest possible EXA connection consist of a hostname followed by a colon and a port number. Instead of the hostname you can also use an IP or IP range.

> The method `getExaConnectionAddress()` gives you the address part that you need to create such a connection to the Exasol instance running inside the test container from the host.

Reviewer: ```suggestion
The method `getExaConnectionAddress()` returns the address part that you need in order to create an EXA connection to the Exasol instance running inside the test container from the host.
```

Reviewer: I find the `EXA connection` to `such connection` more clearer, please feel free to include or dismiss it.
